.gitignore

```diff
@@ -1,7 +1,6 @@
 .idea/
-node_modules/
+.vscode/
+/config/
```

Comment:
Are you sure git is ignoring everything present under config file? It stopped ignoring for me.

Reply:
Yes, its ignoring everything under `config` folder, I confirmed this based on `git status` and based on `sourcetree` too.

Here's my version of `git`:
```
git --version
git version 2.11.0 (Apple Git-81)
```

Reply:
Mine is the same. Strange.